src/plugins/data_views/common/data_views/data_views.ts

```diff
@@ -543,7 +543,7 @@ export class DataViewsService {
       metaFields,
       type: options.type,
       rollupIndex: options.rollupIndex,
-      allowNoIndex: options.allowNoIndex,
+      allowNoIndex: options.allowNoIndex ?? true,
```

Comment:
@mattkime FYI

Reply:
I think since these are private functions we can just permanently set these to true.